sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/DefaultServiceSpecTest.java

```diff
@@ -83,7 +83,8 @@ public void validSimple() throws Exception {
         File file = new File(classLoader.getResource("valid-simple.yml").getFile());
         DefaultServiceSpec serviceSpec = DefaultServiceSpec.newGenerator(RawServiceSpec.newBuilder(file).build(), flags).build();
         Assert.assertNotNull(serviceSpec);
-        Assert.assertFalse(DefaultService.serviceSpecRequestsGpuResources(serviceSpec));
```

Comment:
Do we have a test of a service spec for which `serviceSpecRequestsGpuResources()` does not return `false`?